### ISSUE TITLE
IIFE "Variable declared more than once"

### ISSUE THREAD

**Issue by tomasmcm** [ampproject/rollup-plugin-closure-compiler#219]
## What's the issue?

When compiling a Rollup Svelte project with `format: 'iife'` and `name: 'app'` there is an error `ERROR - [JSC_VAR_MULTIPLY_DECLARED_ERROR] Variable app declared more than once.`

## How do we reproduce the issue?

Rollup config:
```
import svelte from 'rollup-plugin-svelte';
import resolve from 'rollup-plugin-node-resolve';
import commonjs from 'rollup-plugin-commonjs';
import compiler from '@ampproject/rollup-plugin-closure-compiler';

export default {
	input: 'src/main.js',
	output: {
		sourcemap: true,
		format: 'iife',
		name: 'app',
		file: 'public/bundle.js'
	},
	plugins: [
		svelte({
			css: css => {
				css.write('public/bundle.css');
			}
		}),

		resolve({
			browser: true,
			dedupe: importee => importee === 'svelte' || importee.startsWith('svelte/')
		}),
		commonjs(),

		compiler({
			compilationLevel: 'ADVANCED',
			languageOut: 'ECMASCRIPT5_STRICT',
			warningLevel: 'VERBOSE',
		})
	],
};
```

Run the command
`rollup -c`

## Additional context

Changing the format to `format: 'umd'` compiles successfully.
Any ideas where the issue might be?

**Comment by kristoferbaxter** [comment]
Could you attach the source before compiler sees the output? (remove `compiler` from rollup and run once, then attach to this bug).

**Comment by kristoferbaxter** [comment]
Created a gist here: (https://gist.github.com/kristoferbaxter/aa44140f4186d5b1236d9a11f3b27def)

Wasn't able to reproduce the issue when using Closure Compiler directly. What am I missing?

**Comment by tomasmcm** [comment]
It seems the issue is when running svelte and closure compiler one after the other. When I run rollup with just svelte it works. If I copy the output and then run rollup with only closure it also works. When they are both in the plugins (first svelte, then closure) it gives this error.
Maybe something to do with the way the code is passed between plugins?

**Comment by kristoferbaxter** [comment]
Interesting, will take a look at the interaction.

**Comment by kristoferbaxter** [comment]
@tomasmcm – Can you provide a simple repo with all the necessary source to reproduce? I'd like to take a look at this soon.

**Comment by tomasmcm** [comment]
@kristoferbaxter 🙏 https://gist.github.com/tomasmcm/c414c4d2ec98a17f0449be311669b01a
This is all you should need to replicate the issue

**Comment by kristoferbaxter** [comment]
Resolved with #254, released with `0.16.0`.
